.github/workflows/ci.yaml

```diff
@@ -30,3 +30,12 @@ jobs:
       run: |
         pip install  .
         make test
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install dependencies
+      run: pip install .[dev]
```

Comment:
I see pylint mentioned in `make install-dev` , and in `extras_require` in setup.py . It looks like two alternative methods to install a single library. We should probably standardize on one way. I'm not sure yet on which, so also not requesting any changes here. Just pointing it out loud, so everyone knows that could be improved in future.